### ISSUE TITLE
module 'keras.optimizers' has no attribute 'Adam'

### ISSUE THREAD

**Issue by johnnytam100** [run-ai/runai#32]
```
import runai.ga.keras

# Gradient Accumulation
STEPS = 10
optimizer = runai.ga.keras.optimizers.Adam(steps=STEPS)

module 'keras.optimizers' has no attribute 'Adam'
```

Hi, I would really want to use runai for Gradient Accumulation.
However, I met the error "module 'keras.optimizers' has no attribute 'Adam'"
Do you know how to solve this issue?
Thanks!

**Comment by levosos** [comment]
Hi @johnnytam100 and thanks for reaching out!
What is the version of TensorFlow you are using?
Unfortunately we don't support version 2.0 or newer versions.
You can see my explanations in other issues (for example [here](https://github.com/run-ai/runai/issues/21#issuecomment-668095434)).
